docker-compose.yml

```diff
@@ -27,6 +26,15 @@ services:
     command: ["./wait-for-elasticsearch.sh", "elastic:9200", "npm", "run", "update"]
   elastic:
     image: docker.elastic.co/elasticsearch/elasticsearch:6.7.2
-#db: 
-  #image: rethinkdb:2.3.6 
-  # NB: https://github.com/PathwayCommons/factoid/issues/452
+    volumes:
+      - esdata:/usr/share/elasticsearch/data
+  # db: # NB: https://github.com/PathwayCommons/factoid/issues/452 
+  #   image: rethinkdb:2.3.6 
+  #   volumes:
+  #     - dbdata:/data
+
+volumes:
+  esdata:
+  # dbdata:
```

Comment:
Why is dbdata commented out?

Reply:
That bug #452 seems to cripple the app.

Reply:
If you can reproduce it every time with that docker setup, then it would be a good opportunity to try to find a workaround for it.  I can't reproduce the issue reliably on my machine.

Reply:
It happens on the ci server (`...128`) - i set up a mini docker-compose file in `/home/baderlab/tmp`.

Reply:
addressed by #482